tools/figma-inspector/backend/code.ts

```diff
@@ -1,11 +1,12 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
 
+import { listenTS, updateUI } from "./utils/code-utils.js";
 import { generateSlintSnippet } from "./utils/property-parsing.js";
 
 if (figma.editorType === "dev" && figma.mode === "codegen") {
     figma.codegen.on("generate", async ({ node }) => {
-        const slintSnippet = await generateSlintSnippet(node);
+        const slintSnippet = generateSlintSnippet(node);
```

Comment:
Did the function change to not return a promise anymore?

Reply:
ACK, I See that the function doesn't return a promise, so this makes sense and probably didn't work before :)